src/libstd/os/linux/raw.rs

```diff
@@ -93,11 +93,20 @@ mod arch {
     use os::raw::{c_long, c_ulong};
     use os::unix::raw::{gid_t, uid_t};
 
+    #[cfg(target_env = "musl")]
+    #[stable(feature = "raw_ext", since = "1.1.0")] pub type blkcnt_t = i64;
```

Comment:
I'm not sure what stability level this should have. AFAIK, these types are going to be deprecated in the future.